selene/src/standard_library.rs

```diff
@@ -59,39 +63,55 @@ fn from_name<V>(
     config: &CheckerConfig<V>,
     standard_library_name: &str,
     directory: &Path,
+    config_dir: &Option<PathBuf>,
 ) -> color_eyre::Result<Option<StandardLibrary>> {
-    let mut library: StandardLibrary;
+    let mut library: Option<StandardLibrary> = None;
 
-    let toml_file = directory.join(format!("{standard_library_name}.toml"));
-    if toml_file.exists() {
-        let content = fs::read_to_string(&toml_file)?;
+    let mut dirs = vec![directory];
```

Comment:
Full name please, `directories`

Reply:
I changed this and also all the instances of `config_dir` to `config_directory`.

Reply:
Although maybe I should've left `config_dir` since you're already using `current_dir`. 🤔